test/extensions/filters/network/sni_dynamic_forward_proxy/proxy_filter_test.cc

```diff
@@ -99,14 +105,14 @@ TEST_F(SniDynamicProxyFilterTest, LoadDnsCache) {
 
 TEST_F(SniDynamicProxyFilterTest, LoadDnsCacheFromFilterState) {
```

Comment:
Add a test to make sure we fallback on the configured port when the `"envoy.upstream.dynamic_host"` is not set?

Reply:
done